aptos-move/aptos-vm/src/move_vm_ext/write_op_converter.rs

```diff
@@ -99,23 +99,23 @@ impl<'r> WriteOpConverter<'r> {
 
         let maybe_existing_metadata = state_value_metadata_result.map_err(|_| {
             VMStatus::error(
-                StatusCode::STORAGE_ERROR,
+                StatusCode::DELAYED_FIELDS_SPECULATIVE_ABORT_ERROR,
```

Comment:
Why delayed fields here? This is just metadata?

Reply:
oh it will be None if metadata doesn't exist yet?